### PR TITLE
Allow subfolders inside postsPath

### PR DESCRIPTION
**Pull request by danigb**
Hello 👋 

First of all, thanks a lot for the minimal-blog theme 🙏 It's a wonderful theme!

## Motivation

I've make this PR because I wanted to have my posts organized by subfolders. Something like this:
```
content/
  posts/
  stories/
  events/
  drafts/
```

I tried to make the less thing possible to work. But I'm open to other solutions 😅 

## What this PR do

It adds a new (optional) configuration option `additionalPostsFolders` with an array of subfolder names to be searched for posts.

## Usage

In the `gatsby-config.js`:

```
  plugins: [
    {
      resolve: `@lekoarts/gatsby-theme-minimal-blog`,
      options: {
        basePath: `/`,
        blogPath: `/blog`,
        postsPath: `content`,
        additionalPostsFolders: [`posts`, `stories`, `events`, `drafts`],
        ...
```

### CODE REVIEW

File: themes/gatsby-theme-minimal-blog-core/gatsby-node.js [hunk 1]
```diff
@@ -193,7 +193,7 @@ exports.onCreateNode = ({ node, actions, getNode, createNodeId, createContentDig
   const source = fileNode.sourceInstanceName
 
   // Check for "posts" and create the "Post" type
-  if (node.internal.type === `Mdx` && source === postsPath) {
+  if (source === postsPath || additionalPostsFolders.includes(source)) {
```
Comment (danigb) [inline themes/gatsby-theme-minimal-blog-core/gatsby-node.js]: I remove the check about `type` because it's already made at line 185

**Comment by LekoArts** [comment]
Hi, thanks for using the theme!

Could you please expand on your explanation why that change is needed? The desired behavior is already working, see this codesandbox example: https://codesandbox.io/s/minimal-blog-multiple-posts-folders-53bjo

**Comment by danigb** [comment]
Really!? I couldn't make it work 😂 but let me try again. Thanks a lot! I'll close the PR when I test it 🙋‍♂ 

Thanks

**Comment by danigb** [comment]
I don't know why it didn't worked first time. Thanks for the quick answer and, again, for such good theme 👍 
Closing the PR
